core/modules/memman/MemManReal.cc

```diff
@@ -162,7 +214,7 @@ MemMan::Handle MemManReal::lock(std::vector<TableInfo> const& tables, int chunk)
        // (with global lock held) update statistics, generate a file handle,
        // add it to the handle cache, and return the handle.
        //
```

Comment:
Please replace "lock" in the comments for this function with "map" or something similar.